src/components/Modal/index.tsx

```diff
@@ -1,10 +1,11 @@
 import React, { FunctionComponent, ReactNode } from 'react'
-import Modal from 'react-modal'
-import noScroll from 'no-scroll'
+
 import cx from 'classnames'
+import noScroll from 'no-scroll'
```

Comment:
what is the purpose of this lib?

Reply:
The place of the import just changed due to the linter but it's just what we already used in the modal to disable the scroll in case it's opened